cypress/integration/landing_page_spec.ts

```diff
@@ -0,0 +1,64 @@
+import { LandingPo } from '../support/landing.po';
+
+describe('Landing Page test suite', () => {
+  const landing = new LandingPo();
+
+  it('Successfully loads and displays all content of the ngx-bootstrap LP', () => {
+    landing.navigateTo();
+
+    cy.get('.logo').should('be.visible');
+    cy.get('.header-info').should('be.visible');
+    cy.get('.content-logo').should('be.visible');
+    cy.get('.slogan').should('be.visible');
+    cy.get('.descr').should('be.visible');
+    cy.get('.version').should('be.visible');
+    cy.get('.advantages').should('be.visible');
+  });
+
+  it('Get started button redirects to Getting Started page', () => {
+    const buttonText = 'Get started';
+    const searchedUrl = '/getting-started';
+
+    landing.navigateTo();
+    landing.clickByText('.btn', buttonText);
+
+    cy.url().should('include', searchedUrl);
+  });
+
+  it('Github button is enabled and contains link to ngx-bootstrap repo', () => {
+    const buttonText = 'Github';
+
+    landing.navigateTo();
+
+    cy.get('.btn').contains(buttonText).should('be.enabled');
```

Comment:
use chaining for making several assertions 
https://docs.cypress.io/guides/references/best-practices.html#Creating-%E2%80%9Ctiny%E2%80%9D-tests-with-a-single-assertion
` cy.get('.btn').contains(buttonText).should('be.enabled').and.('have.attr', 'href', landing.githubUrl)`